apparmor.d/groups/pacman/pacman

```diff
@@ -102,8 +107,10 @@ profile pacman @{exec_path} {
   @{bin}/touch                       rix,
   @{bin}/tput                        rix,
   @{bin}/update-ca-trust             rPx,
+  @{bin}/uname                       rPx,
```

Comment:
```suggestion
  @{bin}/uname                       rix,
```

Reply:
Why do you want this change? The `uname` profile exists and rPx, hasn't caused problems for me.

Reply:
In this context that does not matter that much, both would work, so let keep it this way.